lib/optionInterfaces.ts

```diff
@@ -13,10 +26,11 @@ export interface PeerJSOption {
 	secure?: boolean;
 	token?: string;
 	config?: RTCConfiguration;
-	debug?: number;
+	debug?: LogLevel;
```

Comment:
So I can convert this over to string literals like this:
![image](https://github.com/peers/peerjs/assets/40000658/f8dbd434-78a6-4dda-ba98-c618f5cc168d)
 
 Changing to suit this would mean doing this all over the application 
![github_issue_peers](https://github.com/peers/peerjs/assets/40000658/3e58c824-1da5-43fa-9fc9-2dcfa8616696)

Let me know if there is a better option or if you want to see this imlemented.